python/adelphi/tests/unit/test_anonymize.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+from adelphi import anonymize
+from adelphi.anonymize import COLUMN_PREFIX, get_name
+from schema_util import get_schema
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest  # noqa
+
+class TestCqlAnonymize(unittest.TestCase):
```

Comment:
anonymizer unit tests

Reply:
we can expand this to test the command-line params too, for now these gravitate towards the anonymization